index.js

```diff
@@ -10,6 +10,28 @@ function posixify(file) {
 const virtualModules = new Map();
 let index = 0;
 
+let config_file = 'webpack.config.js';
```

Comment:
This should be `configFile` for consistency with the rest of the file.

Reply:
good catch thanks! muscle memory